sys/linux/socket.txt

```diff
@@ -317,6 +321,41 @@ ifr_ifru [
 
 define IFREQ_SIZE	sizeof(struct ifreq)
 
+# https://elixir.bootlin.com/linux/v5.3.11/source/include/uapi/linux/if_vlan.h
+vlan_args {
+	cmd		flags[vlan_cmd, intptr]
+	device1		array[int8, 24]
+	u		vlan_u
+	vlan_qos	int8
+}
+
+vlan_u [
+	device2		array[int8, 24]
+	VID		intptr
+	skb_priority	intptr
+	name_type	intptr
+	bind_type	intptr
+	flag		intptr
+]
+
+vlan_cmd = ADD_VLAN_CMD, DEL_VLAN_CMD, SET_VLAN_INGRESS_PRIORITY_CMD, SET_VLAN_EGRESS_PRIORITY_CMD, GET_VLAN_INGRESS_PRIORITY_CMD, GET_VLAN_EGRESS_PRIORITY_CMD, SET_VLAN_NAME_TYPE_CMD, SET_VLAN_FLAG_CMD, GET_VLAN_REALDEV_NAME_CMD, GET_VLAN_VID_CMD
```

Comment:
Please add `include <uapi/linux/if_vlan.h>` at the top.
I see it worked somehow, but it's still good to have proper includes.

Reply:
Thanks for the heads up, added.